UnityProject/Assets/Scripts/Map/LavaLand/LavaLandRandomGenScript.cs

```diff
@@ -38,7 +38,7 @@ public class LavaLandRandomGenScript : MonoBehaviour
 		[SerializeField]
 		private RandomItemSpot mobPools = null;
 
-		private void Start()
+		private void Awake()
```

Comment:
Is it guaranteed that LavaLandManager's Awake is called prior to this one?

Reply:
Look carefully at LavaLandManager. It uses Awake() too (via parent), in order to instantiate the singleton. Therefore, there might be possibility that this script awakes before the manager awakes (probably hierarchy or scene-dependent). That's why this particular case looks fragile.

Reply:
Changed